apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -66,6 +66,13 @@ const SummaryResponses = ({
     [scriptData, selectedSection]
   );
 
+  const eventData = {
```

Comment:
This looks great. My only suggestion would be to wrap it in `React.useMemo` so that we only recalculate it when `scriptData` changes